velox/functions/prestosql/registration/GeneralFunctionsRegistration.cpp

```diff
@@ -30,6 +31,11 @@ extern void registerElementAtFunction(
 void registerAllSpecialFormGeneralFunctions() {
   exec::registerFunctionCallToSpecialForms();
   VELOX_REGISTER_VECTOR_FUNCTION(udf_in, "in");
+  registerFunction<
```

Comment:
Not sure if you checked how this performs vs. the vectorized, but I guess it should be about the same since they both have a dynamic dispatch for the comparison. Is this hot enough to justify a fast-path for primitives?

Reply:
@pedroerp I haven't explicitly checked that. I assume it is the same given that vector function doesn't have any optimizations. This function is used in a rare case where in-list is not constant. Hence, I don't think it is justified to optimize it.